conftest.py

```diff
@@ -1,27 +1,71 @@
-import os
 from pathlib import Path
 
 import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
 
-from jupyter_scheduler.orm import create_session, create_tables
+from jupyter_scheduler.orm import Base
 from jupyter_scheduler.scheduler import Scheduler
 from jupyter_scheduler.tests.mocks import MockEnvironmentManager
 
-pytest_plugins = ("jupyter_server.pytest_plugin",)
+pytest_plugins = ("jupyter_server.pytest_plugin", "pytest_jupyter.jupyter_server")
 
-HERE = Path(__file__).parent.resolve()
-DB_FILE_PATH = f"{HERE}/jupyter_scheduler/tests/testdb.sqlite"
-DB_URL = f"sqlite:///{DB_FILE_PATH}"
 
-TEST_ROOT_DIR = f"{HERE}/jupyter_scheduler/tests/test_root_dir"
+@pytest.fixture(scope="session")
+def static_test_files_dir():
+    return Path(__file__).parent.resolve() / "jupyter_scheduler" / "tests" / "static"
 
 
 @pytest.fixture
-def jp_server_config(jp_server_config):
+def jp_scheduler_root_dir(tmp_path):
```

Comment:
Can you add typing annotations to the return types of these fixtures? Some of these return `pathlib.Path` objects, and others return strings. Even though IDE intellisense doesn't really work for Pytest fixtures (at least on my VSCode instance), I think indicating the return type would be very helpful to future devs adding tests.

Reply:
Agree. Added type annotations for all fixtures that return Paths, strings, or their tuples. I've also added assertions of `tmp_path` and similar as `Path` type to fixtures. At least on my instance, this makes IDE intellisense properly infer the return types increasing the usability for future devs even more.